tests/id-test.js

```diff
@@ -39,11 +40,12 @@ test('Recreate an B58 String', function (t) {
 
 test('Recreate from a Public Key', function (t) {
   var id = PeerId.createFromPubKey(testId.pubKey)
-  t.ok(id)
+  t.ok(id.pubKey == testId.pubKey)
```

Comment:
ah, now it is a string and not a buffer? Let's make sure to add notes to the README of the format of this values, so later we can use it for things with TLS without having to figure out the format again :)